web/src/web/templates/base.html

```diff
@@ -14,48 +15,48 @@
                     <div class="sidebar__header-title">{% block page_header %}Telemetry Data Mapper{% endblock %}</div>
                 </div>
                 <ul>
-                    <li class="sidebar__item">
+                    <li class="sidebar__item" data-step="1" data-position="right" data-intro="This page!">
                         <a href="{{ url_for('index') }}">
                             <span class="icon-home"></span>
                             <span class="half-margin-left">Home</span>
                         </a>
                     </li>
-                    <li class="sidebar__item">
+                    <li class="sidebar__item" data-step="2" data-position="right" data-intro="Use Search to find XPaths, OIDs, whatever you are looking for more easily with keywords!">
                         <a href="{{ url_for('search_es') }}">
                             <span class="icon-search"></span>
                             <span class="half-margin-left">Search</span>
                         </a>
                     </li>
-                    <li class="sidebar__item">
+                    <li class="sidebar__item" data-step="3" data-position="right" data-intro="Matchmaker helps take a list of datapaths like OIDs or XPaths and find what's been mapped as equivalent. Great for upgrading a list of OIDs to XPaths.">
```

Comment:
Did you mean converting or is upgrading the right word to use here?

Reply:
If we're speaking technically, convert is the right word, but the marketer in me likes upgrade.